packages/db/src/services/DatabaseService.ts

```
@@ -326,7 +326,8 @@ export class DatabaseService {
 
   public async getLapData(): Promise<Lap[]> {
     if (!this.isConnected) {
-      throw new Error("Database not connected");
+      console.warn("Database not connected");
```

ESLint

[error] Unexpected console statement.

(no-console)